index.html

```diff
@@ -92,10 +93,15 @@ <h2 class="titlewords">My Recent Works</h2>
               <li class="tag bootag">bootstrap</li>
               <li class="tag rubytag">Ruby</li>
             </ul>
-            <button type="button" class="probtn">See Project</button>
+            <button type="button" id="project-btn" class="probtn">See Project</button>
+            <!-- POP UP WINDOW -->
           </div>
         </div>
-        <div class="project-cards">
+        <div class="my-projects">
+          
+        </div>
+
+        <!-- <div class="project-cards">
```

Comment:
- [ ] Kindly remove the comments from your code, leaving unnecessary comments in your project make it messy and unreadable for the user and future reviewer